src/Components/Calculator.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] `Required`
- [ ] Amazing work so far!
- [ ] However please enable the disabled linters because it defeats the purpose of adding linters